docs/API.md

```diff
@@ -1671,12 +1670,8 @@ const selectOpts = {
   requestProgress: { Enabled: true },
 }
 
-minioClient.selectObjectContent('bucketName', 'objectName', selectOpts, function (err, res) {
-  if (err) {
-    return console.log('Unable to process select object content.', err.message)
-  }
-  console.log('Success')
-})
+const res = minioClient.selectObjectContent('bucketName', 'objectName', selectOpts)
```

Comment:
I believe we're missing `await` keyword?

Reply:
Yes. thank you :+1: